varsim_multi.py

```diff
@@ -207,6 +241,27 @@ def varsim_multi(reference,
                                 required=False)
     rand_dgv_group.add_argument("--sv_dgv", metavar="DGV_FILE", help="DGV file containing structural variants",
                                 required=False)
+    rand_dgv_group.add_argument("--sv_prop_het", metavar="FLOAT", help="Proportion of heterozygous structural variants",
+                                default=0.6,
+                                type=float)
+    dwgsim_group = main_parser.add_argument_group("DWGSIM options")
+    dwgsim_group.add_argument("--dwgsim_start_e", metavar="first_base_error_rate", help="Error rate on the first base",
+                              default=0.0001, type=float)
+    dwgsim_group.add_argument("--dwgsim_end_e", metavar="last_base_error_rate", help="Error rate on the last base",
+                              default=0.0015, type=float)
+    dwgsim_group.add_argument("--dwgsim_options", help="DWGSIM command-line options", default="", required=False)
+
+    art_group = main_parser.add_argument_group("ART options")
+    art_group.add_argument("--profile_1", metavar="profile_file1", help="ART error profile for first end", default="")
+    art_group.add_argument("--profile_2", metavar="profile_file2", help="ART error profile for second end", default="")
+    art_group.add_argument("--art_options", help="ART command-line options", default="")
+
+    pbsim_group = main_parser.add_argument_group("PBSIM options")
```

Comment:
Let us remove pbsim from the list of supported simulators. I think I had not put it in `varsim_multi.py`, so no need to add it here.

Reply:
Hi @marghoob I have removed simulator-specific options. Thanks.